README.md

```diff
@@ -37,13 +37,23 @@ Fingerprint Pro Angular SDK is an easy way to integrate  **[Fingerprint Pro](htt
 
 ## Table of contents
 
+- [Requirements](#requirements)
 - [Installation](#installation)
 - [Getting started](#getting-started)
 - [Caching strategy](#caching-strategy)
 - [Documentation](#documentation)
 - [Support and feedback](#support-and-feedback)
 - [License](#license)
 
+## Requirements
+
+The following dependencies are required:
+
+- TypeScript >=4.6
+- Node 16+
+- Angular 13+
+- RxJS 7.5+
```

Comment:
Let's remove RxJs mention. I checked [our library package.json](https://github.com/fingerprintjs/fingerprintjs-pro-angular/blob/main/projects/fingerprintjs-pro-angular/package.json) and we haven't it in deps. Just use it as an angular core component.

Reply:
Removed RxJS from list